src/config/graphical-session/graphics-drivers/nvidia.md

```diff
@@ -0,0 +1,79 @@
+# NVIDIA
+
+## nouveau (Open Source Driver)
+
+Install the Mesa DRI driver `mesa-dri` and optionally the Xorg driver
+`xf86-video-nouveau`.
+
+> Note: This is already included in the `xorg` meta-package, but is needed when
+> installing via `xorg-minimal`.
+
+This driver is developed mostly by the community, with little input from Nvidia,
+and is not as performant as the proprietary driver. It is required in order to
+run most Wayland compositors.
+
+## nvidia (Proprietary Driver)
+
+The proprietary drivers are available in the
+[non-free](../../../xbps/repositories/official/nonfree.md) repository.
+
+Check if your graphics card belongs to the [legacy
+branch](https://www.nvidia.com/en-us/drivers/unix/legacy-gpu/). If it does not,
+install the `nvidia` package. Otherwise you should install the appropriate
+legacy driver, either `nvidia390` or `nvidia340`.
+
+| Brand  | Type        | Model                           | Driver Package |
+|--------|-------------|---------------------------------|----------------|
+| NVIDIA | Proprietary | 500+                            | `nvidia`       |
+| NVIDIA | Proprietary | 300/400 Series                  | `nvidia390`    |
+| NVIDIA | Proprietary | GeForce8/9 + 100/200/300 Series | `nvidia340`    |
+
+The proprietary driver integrates in the kernel through
+[DKMS](../../kernel.md#dynamic-kernel-module-support-dkms).
+
+This driver offers better performance and power handling, and is recommended
+where performance is needed.
+
+## 32-bit program support (glibc only)
+
+In order to run 32 bit programs with driver support, you need to install
+additional packages.
+
+For nouveau, install the `mesa-dri-32bit` package.
+
+For nvidia, install the `nvidia<x>-libs-32bit` package. `<x>` represents the
+legacy driver version (340 or 390) or can be left empty for the main driver.
+
+## Reverting from nvidia to nouveau
+
+### Uninstalling nvidia
+
+In order to revert to nouveau, install the nouveau driver (if it was not
+installed already), then remove `nvidia`, `nvidia390` or `nvidia340`.
+
+> `nvidia340` does not support `libglvnd`, so you must install it after removing
```

Comment:
Remove blockquote here.

"If you were using the \`nvidia340\` driver, you will need to install the \`libglvnd\` package after removing the \`nvidia340\` package."